CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Rust IPFS
+
+Welcome, and thank you for your interest in contributing to Rust IPFS. Issues and pull requests are encouraged.
+
+## Target Build
+
+Rust IPFS will always target the current _stable_ version of Rust that is released. Our CI/CD tests will reflect this.
+
+TODO: Instructions on installing the up-to-date stable toolchain
+TODO: First Principles
+
+## Roadmap
+
+- Link to roadmap issue or project?
+
+## Contributing
+
+Reporting Bugs
+Suggesting Enhancements
+Your First Code Contribution
+Pull Requests
+
```

Comment:
I'd like to add here "documentation fixes and improvements" or similar. I think "open for doc prs" is quite important as a sign of appreciating such. I think that while there is little documentation at the moment, welcoming "usage issues" as could be a good idea. Though, in general I go to this file when wanting precise information on the tactics of the project, how exactly should I fill the bug report in. Perhaps my ideas are more geared towards README?